packages/nitro-protocol/package.json

```diff
@@ -2,7 +2,7 @@
   "name": "@statechannels/nitro-protocol",
   "description": "A protocol for state channel networks",
   "version": "0.1.0",
-  "author": "George Knee",
+  "author": "statechannels",
```

Comment:
I think it's better to list specific people. It can be an array.

Reply:
This matches our other published package (devtools). It's an open source project so I think a list of authors would be a bit of a pain to maintain.